apps/dashboard/src/components/primitives/editor.tsx

```diff
@@ -60,5 +151,7 @@ export const Editor = ({ value, placeholder, className, height, size, onChange }
     }
   }, [setContainer]);
 
+  // TODO remove default HTML editor tags
```

Comment:
leftover, I wasn't able to remove these tags they come with the liquid package :/